php/src/Google/Protobuf/Syntax.php

```diff
@@ -4,6 +4,8 @@
 
 namespace Google\Protobuf;
 
+use UnexpectedValueException;
```

Comment:
I got warning: The use statement with non-compound name 'UnexpectedValueException' has no effect
Remove that?

Reply:
I found that I got this warning only for the no-namespace enum. Is that what you found? I have added a commit to remove the use statement from the no-namespace case.

Reply:
yes. For NoNamespace enum